server/services/caldav/lib/calendar/calendar.requests.js

```diff
@@ -22,7 +22,7 @@ function getElementsByTagRegex(container, regex) {
  * requestCalendars(xhr, homeUrl)
  */
 async function requestCalendars(xhr, homeUrl) {
-  const ICAL_OBJS = new Set(['VEVENT', 'VTODO', 'VJOURNAL', 'VFREEBUSY', 'VTIMEZONE', 'VALARM']);
```

Comment:
Is there on impact on calendar events on other timezones? If I receive events in other timezones, does it still work?

Reply:
This part check only if calendars contains contents displayable in calendar (VEVENT, VFREEBUSY...). With these changes it can contain VTIMEZONE but we synchronize it only if it contains also a VEVENT or a VFREEBUSY. The error was that Deck Nextcloud app create calendar with only VTODO compatibility and same if we have a calendar only with VTIMEZONE we don't need to sync it because we will have nothing to display